Detector/DetCRD/compact/CRD_common_v01/DC_Simple_v01_01.xml

```diff
@@ -61,40 +56,34 @@
   </regions>
 
   <detectors>
-    <detector id="DetID_DC" name="DriftChamber" type="DriftChamber" readout="DriftChamberHitsCollection" vis="BlueVis" sensitive="true" insideTrackingVolume="true" limits="DC_limits">
+    <detector id="DetID_DC" name="DriftChamber" type="DriftChamber" readout="DriftChamberHitsCollection" vis="VisibleBlue" sensitive="true" region="DriftChamberRegion" limits="DC_limits">
       <envelope vis="SeeThrough">
```

Comment:
Hi @myliu-hub ,
As you already add the chamber id, do you need to update this part? Or you already define the default value?
Tao